README.md

```diff
@@ -139,65 +139,61 @@ Figures are in the folder "Results".
 
 
 ## Running the Workflow
+Instructions are provided for running the workflow on a single computer or a [SLURM cluster](https://slurm.schedmd.com/).
 
-This workflow requires R, conda, and docker to run.
+### Software Requirements
+To run the workflow [conda](https://docs.conda.io/projects/conda/en/stable/) and [singularity (aka apptainer)](https://apptainer.org/) must to be installed.
 
-### Installing snakemake
+### Hardware Requirements
+Minimally the workflow requires 1 CPU, 5 GB memory, and 20 GB disk space.
+A Linux machine is required for this workflow to provide Singularity containerization.
 
-To run the workflow we use snakemake.
-See the [official instructions for installing snakemake](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html).
+### Install Workflow Runner
+To run the workflow [snakemake](https://snakemake.readthedocs.io/en/stable/index.html) with [mamba](https://mamba.readthedocs.io/en/latest/) must be installed.
+To handle this we create a new conda environment named "snakemake".
 
-To install snakemake on the OSC cluster run:
-```
-module load miniconda3
-conda create -n snakemake -c bioconda -c conda-forge -c r snakemake r-essentials r-xml -y
-```
+If you are running the workflow on a cluster that provides a conda environment module you should load that module
+(eg. `module load miniconda3`).
 
-where `-n` designates the name, "snakemake", and `-c` designates the channel(s): "bioconda", "conda-forge", and "r".
-
-To check that the environment was made:
-```
-conda info -e
+Run the following command to create a conda environment named "snakemake" with the required workflow dependencies.
+```console
+conda create -c conda-forge -c bioconda -n snakemake snakemake mamba
 ```
+Enter "Y" when prompted to install snakemake and mamba.
 
-### Installing R packages
```

Comment:
Do we need to have a section that says it's a container now?

Reply:
You mean a `it's a conda environment now`?

Reply:
I added a note about conda and singularity environments:
https://github.com/hdr-bgnn/Minnow_Segmented_Traits/blob/afc897c631d4177d7a6591184f5480c51ca4a42d/README.md?plain=1#L146-L152